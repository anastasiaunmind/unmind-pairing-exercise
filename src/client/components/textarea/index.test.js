import React from "react";
import { render } from "@testing-library/react";
import Textarea from ".";

test("renders textarea", () => {
    const { container } = render(<Textarea placeholder="placeholder here" rows={10} text="" onChange={() => {}} />);

    expect(container).toMatchSnapshot();
});
