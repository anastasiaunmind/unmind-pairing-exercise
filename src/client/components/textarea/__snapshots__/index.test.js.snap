// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders textarea 1`] = `
<div>
  <textarea
    class="sc-bdnxRM eJmVDo"
    placeholder="placeholder here"
    rows="10"
  />
</div>
`;
